lib/datasets/japanese-english-bilingual-corpus.rb

```diff
@@ -0,0 +1,271 @@
+require "rexml/streamlistener"
+require "rexml/parsers/baseparser"
+require "rexml/parsers/streamparser"
+require "time"
+
+require_relative "dataset"
+require_relative "tar-gz-readable"
+
+module Datasets
+  class JapaneseEnglishBilingualCorpus < Dataset
+    include TarGzReadable
+    Article = Struct.new(:id,
+                         :copyright)
+
+    Title = Struct.new(:article_id,
+                      :japanese,
+                      :translate_ver1,
+                      :translate_ver1_comment,
+                      :translate_ver2,
+                      :translate_ver2_comment,
+                      :check_ver1,
+                      :check_ver1_comment)
```

Comment:
How about just using `check_ver1` as `:english`? It seems that others aren't useful information.